org.spoofax.jsglr2/src/main/java/org/spoofax/jsglr2/imploder/TermTreeFactory.java

```diff
@@ -27,9 +29,9 @@ public TermTreeFactory(ITermFactory termFactory) {
     }
 
     @Override public IStrategoTerm createNonTerminal(String sort, String constructor, List<IStrategoTerm> childASTs,
-        IToken leftToken, IToken rightToken) {
+            IToken leftToken, IToken rightToken) {
```

Comment:
A few months ago, all files in this project were formatted using the settings from https://github.com/metaborg/spoofax-deploy/tree/master/eclipse/style , which seems to have four spaces as continuation indent. Is (y)our setup incorrect?

Reply:
I fixed the formatting to be consistent with the rest.